requirements-base.txt

```diff
@@ -66,7 +66,7 @@ rfc3986-validator>=0.1.1
 # [end] jsonschema format validators
 sentry-arroyo>=2.16.5
 sentry-kafka-schemas>=0.1.109
-sentry-ophio==0.2.7
+sentry-ophio==0.2.9
```

Comment:
We had to do two releases:
https://github.com/getsentry/ophio/pull/63
https://github.com/getsentry/ophio/pull/64